### PR TITLE
create a test sample

### PR DESCRIPTION
**Pull request by heming-ma**
# What

# Why

# How

### CODE REVIEW

**Review by jw8823 — APPROVED** [review]

File: Collimate Mobile/Data/ModelDAO/Chats.swift [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Chats: NSObject {
 		let chat = Chat()
 
         chat.chatId	= chat.objectId
+        hghg
```
Comment (SteveSun12138) [inline Collimate Mobile/Data/ModelDAO/Chats.swift:24]: hljkhj

**Review by SteveSun12138 — APPROVED** [review]
aaa

**Review by MichaelXiaoKun — COMMENTED** [review]
heiheihei

**Comment by zsxing99** [comment]
Closed pr due to test review

**Pull request closed without merging**